test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -94,55 +96,44 @@ var _ = BeforeSuite(func() {
 	}
 	masterSSHPrivateKeyFilepath = cfg.GetSSHKeyPath()
 	longRunningApacheDeploymentName = "php-apache-long-running"
+	kubeConfig, err = GetConfig()
```

Comment:
We're now re-using a common `kubeConfig` instance across all tests (kubeconfig data representation is static from the perspective of a single test run)